### DISCUSSION TITLE
我在进行bt下载时已经设置了SRC-IP-CIDR和IP-CIDR，为什么流量还在走代理

### DISCUSSION THREAD

**Discussion by yhw2003** [vernesong/OpenClash#2973] [Q&A]
<img width="600" alt="image" src="https://user-images.githubusercontent.com/46086392/215950030-7944a5b1-e19e-4565-86ff-71a774de9f97.png">

**Comment by techstay** [comment]
这里显示的应该是经过clash内核处理的流量，并不一定是走代理吧。判断是否走代理，用yacd面板，看看应用的规则是直连还是代理
